jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/newcase/NewCaseInstancePresenter.java

```diff
@@ -104,37 +96,37 @@ protected void loadCaseDefinitions() {
     protected void createCaseInstance(final String caseDefinitionName,
                                       final String owner,
                                       final List<CaseRoleAssignmentSummary> assignments) {
-        final CaseDefinitionSummary caseDefinition = caseDefinitions.get(caseDefinitionName);
-        if (caseDefinition == null) {
+        final Optional<CaseDefinitionSummary> cds = getCaseDefinitionSummary(caseDefinitionName);
```

Comment:
Can we rename _caseDefinitionName_ argument to _caseDefinitionId_ so it's consistent with naming of arguments in _loadCaseRoles()_ and _getCaseDefinitionSummary()_?